scripts/docker/Dockerfile.sycl

```diff
@@ -10,6 +10,7 @@ RUN apt-get update && apt-get install -y \
         ninja-build \
         python3 \
         git \
+        libomp-dev \
```

Comment:
Do you still need this when you are only building for the `dpcpp` backend? Note, that this build doesn't even support `OpenMP` to be enabled in `Kokkos` AFAIK.

Reply:
Without it, I get a linker error
```
[ 11%] Linking CXX executable KokkosCore_UnitTest_CMakePassCmdLineArgs
/usr/bin/ld: cannot find -lomp
clang-14: error: linker command failed with exit code 1 (use -v to see invocation)
core/unit_test/CMakeFiles/KokkosCore_UnitTest_CMakePassCmdLineArgs.dir/build.make:105: recipe for target 'core/unit_test/KokkosCore_UnitTest_CMakePassCmdLineArgs' failed
make[2]: *** [core/unit_test/KokkosCore_UnitTest_CMakePassCmdLineArgs] Error 1
CMakeFiles/Makefile2:1890: recipe for target 'core/unit_test/CMakeFiles/KokkosCore_UnitTest_CMakePassCmdLineArgs.dir/all' failed
make[1]: *** [core/unit_test/CMakeFiles/KokkosCore_UnitTest_CMakePassCmdLineArgs.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....

```